packages/retail-ui-extensions/src/extension-api/standard-api.ts

```diff
@@ -1,11 +1,13 @@
 import type {CartApi} from './cart-api';
 import type {LocaleApi} from './locale-api';
 import type {NavigationApi} from './navigation-api';
+import type {SmartGridApi} from './smartgrid-api';
 import type {SessionTokenApi} from './session-token-api';
 
 export type StandardApi<T> = {[key: string]: any} & {
   extensionPoint: T;
 } & LocaleApi &
   CartApi &
   NavigationApi &
```

Comment:
Let's also remove this from the StandardAPI, and only include it in the other [extension point ](https://github.com/Shopify/ui-extensions/blob/6a3fbaa0d197fc11a4bf8a6e143b9215228171dd/packages/retail-ui-extensions/src/extension-points/extension-points.ts#L13)